src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Internship Experience with Layer5 - Sudhanshu Dasgupta"
+subtitle: "Unleashing the Journey: From Open-Source Enthusiast to Internship Success"
+date: 2023-06-15 10:30:05 -0530
+author: Sudhanshu Dasgupta
+category: "Community"
+description: "This is a blog sharing my internship and open-source experience with Layer5"
+category: Community
+tags:
+ - Community
+ - Internship
+ - Layer5
+type: Blog
+published: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+
+
+<BlogWrapper>
+
+<p>
+Hello👋, I'm Sudhanshu Dasgupta, a driven individual from the quaint city of Katni in Madhya Pradesh, India. 
+I have an unwavering passion for learning and exploring new horizons in the world of technology and very passionate for open-source. Currently, I am pursuing my 
+studies at ITM Gwalior, with a focus on building a successful career in Computer Science and Engineering.
+</p>
+
+<h2 id="journey">"When and How did it all Start?" ✈️</h2>
+<h3>Let's uncover the Origin Story</h3>
+
+<p>
+It all began at the start of the year when I made a resolute decision to dive into the world of open-source. Eager to contribute and make a difference, I actively participated in various communities, seeking opportunities to learn and grow.
+Amidst my exploration for exciting projects to contribute to, I stumbled upon Layer5. The moment I delved into their endeavors, I was captivated by the diverse range of tech-stack-driven initiatives they were involved in. Without hesitation, I knew that this was the perfect place for me to start.
+
+Venturing forth, I attended my very first Newcomers meet, a gateway to understanding the vibrant community and its wide array of projects. It was an enriching experience, immersing myself in the ethos of collaboration and knowledge sharing that defined the community's spirit.
+Taking a small yet significant step forward, I eagerly embraced a task assigned to me.
+I understood that even the tiniest actions can pave the way for monumental achievements. But my journey didn't halt there; fueled by my unwavering commitment, continuous skill development, and active engagement with the community, I witnessed remarkable progress.
+And then, within a mere two months, my dedication and contributions were acknowledged, leading to an incredible milestone—I was awarded an internship at Layer5.
+</p>
+
+<h2 id="experience">
+ My Experience of Internship with Layer5🚶
+</h2>
+
```

Comment:
@sudhanshutech Let's add some images if you prefer.

Reply:
sure @Chadha93